gnd/src/main/java/com/google/android/gnd/ui/projectselector/ProjectSelectorViewModel.java

```diff
@@ -61,7 +61,7 @@
     disposeOnClear(
         dataRepository
             .getProjectSummaries(user)
-            .subscribe(projectSummaries::setValue, this::onProjectSummariesError));
+            .subscribe(projectSummaries::postValue, this::onProjectSummariesError));
```

Comment:
This had to be modified because the db calls are scheduled in background thread. `postValue` allows async calls while `setValue` doesn't.

Reply:
Good call, thanks!